### DISCUSSION TITLE
Why factor of 10000 in weight?

### DISCUSSION THREAD

**Discussion by ram1123** [vukasinmilosevic/Fiducial_XS#15] [Q&A]
Hi @tjavaid & @qyguo 

Do you know why in the weight the factor of 10000 is added?

https://github.com/vukasinmilosevic/Fiducial_XS/blob/cb0c808b6c9148dd227dac4e3f006685ff2611b7/efficiencyFactors.py#L212-L213

**Comment by qyguo** [comment]
It is always recoweight = "genWeight*pileupWeight*dataMCWeight" in our code. Maybe we need to know when we would apply the case recoweight = "totalWeight". In Tahir's original code I see the case but commented. This may be helpful for us to understand it. https://github.com/tjavaid/Fiducial_XS/blob/c9234906f4f11978a676d5588ef2ad8bd9360af5/FidXS_13TeV_2018_80X_preM19_newBinning_preApp/efficiencyFactors.py#L141.
By the way, the sumw[sample] is the hist stored in our Ntuple TH1F sumWeights. It is filled by histContainer_["SUMWEIGHTS"]->SetBinContent(1,sumWeightsTotal); sumWeightsTotal += genWeight.
